lib/rets/metadata/lookup_type.rb

```diff
@@ -4,8 +4,8 @@ class LookupType
       attr_reader :long_value, :value
 
       def initialize(lookup_type_fragment)
-        @value       = lookup_type_fragment["Value"]
-        @long_value  = lookup_type_fragment["LongValue"]
+        @value       = lookup_type_fragment["Value"].rstrip
+        @long_value  = lookup_type_fragment["LongValue"].rstrip
```

Comment:
I double-checked with a script and `strip` doesn't affect any types that aren't affected by `rstrip` so it seems safe to use either.